pkg/metrics/metrics.go

```diff
@@ -14,6 +14,16 @@ const (
 	Success   = "success"
 )
 
+var authenticatorMetrics Metrics
+
+func InitMetrics(registerer prometheus.Registerer) {
```

Comment:
could we try using an init function to avoid calling `InitMetrics` at all? Test and prod code will benefit from not having to call InitMetrics. Let me know how that looks

Reply:
> so we need a way to clear the metrics after each test run

How do we clean up today?

Reply:
We don't need to because [we create](https://github.com/kubernetes-sigs/aws-iam-authenticator/blob/master/pkg/server/server_test.go#L104-L109) a new metrics namespace for each test setup.